src/wallet/rpcwallet.cpp

```diff
@@ -1792,6 +1810,8 @@ UniValue listsinceblock(const JSONRPCRequest& request)
         filter = filter | ISMINE_WATCH_ONLY;
     }
 
+    bool include_removed = (request.params.size() < 4 || request.params[3].get_bool());
```

Comment:
When adding RPCs or arguments, good advice is to treat null arguments the same as missing arguments: https://github.com/bitcoin/bitcoin/pull/10143#discussion_r112140887

I think that means this line would become:

`bool include_removed = (request.params.size() < 4 || request.params[3].isNull() || request.params[3].get_bool());`